src/MinimumNodeService.cpp

```diff
@@ -178,10 +178,6 @@ void MinimumNodeService::process(const Action *action)
   heartbeat();
 }
 
-// TODO: This list is used while implementing MNS. Remove once done.
-// MNS shall implement these opcodes in incoming requests
-// * RDGN - Request Diagnostic Data (0x87)
```

Comment:
RDGN is now implementd in MinimumNodeServiceWithDiagnostics. This comment should have been removed when that class was created.